beacon-light-client/plonky2/crates/circuits/src/deposits_accumulator_balance_aggregator/first_level.rs

```diff
@@ -899,7 +899,7 @@ mod test {
         let proof = circuit.prove(pw).unwrap();
         println!("Proof generated in {:?}", s.elapsed());
 
-        let result =
+        let _result =
```

Comment:
Shouldn't there be some sort of an assertion?

Reply:
We are just leaving these circuits for future reference but they are not going to be used